composer.json

```diff
@@ -49,11 +49,11 @@
 		"mediawiki/semantic-media-wiki": "~3.0",
 		"nicmart/tree": "^0.2.7",
 		"data-values/geo": "~4.0|~3.0|~2.0",
-		"phpoffice/phpspreadsheet": "^1.4",
 		"symfony/css-selector": "^3.3",
 		"mediawiki/mermaid": "~2.1"
 	},
 	"suggest": {
+		"phpoffice/phpspreadsheet": "Required for 'format=spreadsheet'",
 		"phpoffice/phpexcel": "Required for 'format=excel'",
```

Comment:
This can probably go since "excel" is meant to be an alias of "spreadsheet", meaning that the old "phpexcel" lib is not needed in either case.